examples/typescript-plugin/README.md

```diff
@@ -0,0 +1,62 @@
+# Typescript Plugin Example
+
+This is an example plugin written in Typescript that serves as a base for more complex projects.
+
+## Requirements
+
+- A Strapi application running in V4 or above
+- Node
+
+## Get Started
+
+### Building the Plugin
+
+In order to use the plugin, you'll need to transpile the code to Javascript.
+
+To do so, you can use one of the following commands:
+
+1. `yarn run build`
+
+   Which will transpile the plugin's code once
+
+2. `yarn run build:watch`
+
+   Which will transpile the plugin's code everytime a change is detected
+
+You can configure the build directory from your `tsconfig.json` file using `compilerOptions.outDir`.
+
+### Linking the Plugin
+
+In order for Strapi to recognize & use the plugin, you'll need to register it.
+
+Head over to the `/config/plugin.js` file located in your Strapi app and adds the following configuration (don't forget to replace `resolve`'s path with the one to your transpiled plugin code):
+
+```js
+module.exports = () => ({
+  // ...
+  tsplugin: {
+    enabled: true,
+    resolve: `<ts_plugin_dir>/<out_dir>`, // From the root of the app
```

Comment:
Could be nice to write a concrete example of path, wdyt?

Reply:
What do you have in mind? It'll change for every app :confused: 
I thought about putting smth like "`plugins/typescript-plugin/build`" at first, but it could be misleading in the end. WDYT?

Reply:
Yes I was thinking of that :) It's nice to see you point to the build folder and not a specific file or else